fe/fe-core/src/main/java/com/starrocks/alter/Alter.java

```diff
@@ -641,9 +644,14 @@ public void processAlterTable(AlterTableStmt stmt) throws UserException {
                 } else if (properties.containsKey(PropertyAnalyzer.PROPERTIES_REPLICATED_STORAGE)) {
                     ((SchemaChangeHandler) schemaChangeHandler).updateTableMeta(db, tableName, properties,
                             TTabletMetaType.REPLICATED_STORAGE);
+                } else if (properties.containsKey(PropertyAnalyzer.PROPERTIES_BINLOG_ENABLE) ||
+                        properties.containsKey(PropertyAnalyzer.PROPERTIES_BINLOG_TTL) ||
+                        properties.containsKey(PropertyAnalyzer.PROPERTIES_BINLOG_MAX_SIZE)) {
+                    ((SchemaChangeHandler) schemaChangeHandler).updateBinlogConfigMeta(db, olapTable.getId(),
```

Comment:
Exception will be thrown if FE meta is modified successfully, and what the result of alter?

Reply:
it should fail, but will eventually be consistent through the reporthandler mechanism

Reply:
But FE meta is modified successfully, why failed?